content/get-involved/report-issues/how-issues-are-sorted.md

```diff
@@ -0,0 +1,100 @@
+
+A major issue affects a large percentage of users (> 30%) and matches at least one of the following:
+
+- It impacts law compliance
+- It has a strong impact on the usability of the front-office / back-office or blocks another project
+- It is an important problem but not necessarily blocking the main activity of the seller
+
+Examples:
+
+- Being unable to add, configure or delete a theme or a module
+- Difficulty in operating a module properly
+- Impacts the price the customer pays
+
+### Minor
+
+The bug affects minor functionality or non-critical data and there is a reasonable workaround, even if it can be annoying when using your shop.
+
+
+Examples:
+
+- A tolerable slowdown
+- A display problem that prevents users from doing something non-critical (eg: can’t click on an element that can be accessible in another way)
+- An error message displayed in your back-office that can be dismissed
+- Cloning a product doesn't copy all of it's data
+- Inaccurate statistics
+
+
+### Trivial
+
+The bug doesn’t affect any functionality or data. It does not impact productivity or efficiency. It is only an inconvenience without functional impact and it does not even need a workaround.
+
+Examples:
+
+- Cosmetic issues
+- Wrong translation in a specific language: that can be solved on [Crowdin](https://crowdin.com/project/prestashop-official)
+- Missing confirmation message after an action
+- A link opened in the same tab instead of a new tab
+
+
+## Issue Prioritization
+
+Assessing severity helps to prioritize issues but it is not the only criterion at stake. Given two equally severe issues, how to choose one over the other ?
+
+Prioritization is done by representatives of the Development team, the Product Management team, and the Quality Assurance team.
+
+Together, during regular meetings, they look at the new confirmed issues and they sort them.
+
+In order to make sure that a given bug does not damage PrestaShop's image nor it affects the confidence merchants can have in PrestaShop, they take special care and strive to make every version of PrestaShop better than the one before. Since no one wants to introduce new bugs while fixing other bugs, regressions (new bugs created accidentally when fixing or improving an existing feature) are usually prioritized higher than older bugs. By doing this, the overall software stability is ever increasing.
+
+Then the issue's technical complexity is also studied : that is, whether the bug is easy or complex to fix.
+
+Sometimes a smaller bug will be prioritize over a bigger one. This is because a complex bug may require big technical changes which are not suitable until a later version. This may be because a would require applying backwards-incompatible changes (which are bad for module developers), or because it can be better addressed as a part of a larger project – there is no use fixing a bug if the whole feature is due to be revamped in the near future.
```

Comment:
```suggestion
Sometimes a smaller bug will be prioritized over a bigger one. This is because a complex bug may require big technical changes which are not suitable until a later version. This may be because it would require applying backward-incompatible changes (which are bad for module developers), or because it can be better addressed as a part of a larger project – there is no use fixing a bug if the whole feature is due to be revamped in the near future.
```